## Cont. from L10
	- ![ðŸ–¼ Slide1.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203622_Slide1.PNG)
		- example #HPC #code #SLURM
		-
	- ![ðŸ–¼ Slide2.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203622_Slide2.PNG)
		- #UQ 7
		-
	- ![ðŸ–¼ Slide3.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203622_Slide3.PNG)
		- #FASTQ #[[for loop]]
		- in addition to the two #[[for loop]] snippets shown in the slide, the following format is also correct
			- `for variable in [list]; do command; done`
		- note: use `echo` to print out intermediate results on HPC to check your #code
		-
	- ![ðŸ–¼ Slide4.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203622_Slide4.PNG)
		- #fastp #[[for loop]]
		- Note: Lines 6 and 7 belong to the same line of text (with text wrapping).
		-
	- ![ðŸ–¼ Slide5.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203622_Slide5.PNG) #Slido
		- Answer: all lines 1-9 in the example code are correct
		-
	- ![ðŸ–¼ Slide6.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203622_Slide6.PNG)
	- ![ðŸ–¼ Slide7.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203622_Slide7.PNG)
		- (#SAMtools can be used for filtering alignments based on mapping quality etc. (values of various fields) in SAM/BAM files)
	-
	- ![ðŸ–¼ Slide8.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203623_Slide8.PNG)
	- ![ðŸ–¼ Slide9.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203623_Slide9.PNG)
		- Quality control (#QC) of aligned reads:
			- By reviewing statistics generated by the aligner (e.g., STAR's Log.final.out) to understand the alignment outcome and efficacy.
			- By visualizing the alignment in a genome context using tools like IGV (Integrative Genomics Viewer) to assess alignment quality.
			- (Using tools such as QoRTs and RSeQC to generate visualizations like gene_diversity_plot, read_distribution_plot, and geneBody_coverage_plot to assess various aspects of the alignment quality.)
		-
- # Section 8: Counting aligned reads
	- ![ðŸ–¼ Slide10.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203623_Slide10.PNG)
		- raw #reads -> preprocessing & alignment -> #[[aligned reads]] -> counting #[[assigned reads]] (i.e., those uniquely assigned to genes) -> #counts -> normalization
		-
	- ![ðŸ–¼ Slide11.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203623_Slide11.PNG)
		- #transcripts vs. genes: transcript sequences are not equivalent to gene sequences due to alternative splicing, RNA editing etc., leading to multiple #isoforms
			- Transcript-level analysis is more complex than gene-level due to the existence of multiple isoforms and the complexity of reconstructing the gene model (i.e., #[[transcriptome assembly]] ) from data.
		-
	- ![ðŸ–¼ Slide12.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203623_Slide12.PNG)
		- reads mapping to a single gene are counted, while reads that map to multiple genes are usually discarded to avoid ambiguity in gene expression estimation.
		-
	- ![ðŸ–¼ Slide13.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203623_Slide13.PNG)
	- ![ðŸ–¼ Slide14.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203624_Slide14.PNG)
		- #featureCounts #SLURM
			- featureCounts is a highly efficient program for counting reads mapping to genomic features such as genes or exons.
			- Lines 20 and 32 show how to use featureCounts to handle multiple input BAM/SAM files
				- note that STAR & fastp programs donâ€™t have this native function to handle multiple input files (and thus need a #[[for loop]] )
				-
	- ![ðŸ–¼ Slide15.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203624_Slide15.PNG)
	- ![ðŸ–¼ Slide16.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203624_Slide16.PNG)
		- #featureCounts output files -> #[[counts table]]
			- Note: download the files and then open with Excel to see the counts column(s)
	-
- # Section 9: Counts normalization
	- ![ðŸ–¼ Slide17.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203624_Slide17.PNG)
		- read #counts are not equivalent to gene expression levels.
		- counts #normalization is necessary to eliminate systematic effects on read counts (e.g., differences in library size, RNA composition, batch effects, and other technical variations) that are not associated with gene expression. This allows for proper comparison of read counts across different samples.
		-
	- ![ðŸ–¼ Slide18.PNG](../assets/storages/logseq-plugin-multiple-assets/20240217_203624_Slide18.PNG)