public:: false

-
-
-
- **S20**: RNA electrophoresis result interpretation
	- See note on S22 (for intact, partially degraded, and heavily degraded RNA)
	- mRNA shows a smeared profile (because of the varying sizes of tens of thousands of transcripts)
- S21: Slido question
- S22: RNA quality (#RIN )
	- RNA integrity number (RIN) is a measure of RNA purity and integrity, which is correlated with the 28S:18S rRNA ratio.
	- Electropherograms:
		- Intact RNA is characterized by clear and distinct peaks for 18S and 28S rRNA
		- Partially degraded RNA shows 28S and 18S rRNA peaks but with less definition.
		- Heavily degraded RNA has a smeared profile and a much fainter residual 28S rRNA peak.
		- Completely degraded RNA shows no distinct rRNA peaks.
- **S23-25**: #mRNA-library preparation
	- **Total RNA Extraction from samples**: Total RNA includes all types of RNA within the sample.
	- **mRNA Purification**: mRNA is isolated from total RNA using oligo-dT-based enrichment or by rRNA depletion.
		- mRNA enrichment is done typically using oligo-dT beads which bind to the poly-A tail of mRNA.
		- rRNA depletion can be done using specific probes. It is less common due to higher cost and complexity, but is a preferred method for applications where non-polyadenylated RNA species are of interest.
	- **Fragmentation**: The purified mRNA is fragmented to smaller pieces to create cDNAs with suitable size for sequencing.
	- **Conversion into double-stranded cDNA**
		- **First-Strand cDNA Synthesis**: The fragmented mRNA is reverse transcribed into first-strand cDNA, using the RNA as a template.
		- **Second-Strand Synthesis and Processing**: The first-strand cDNA is converted into double-stranded cDNA (ds-cDNA), which is then end-repaired and adenylated (A-tailing) to prepare for adapter ligation.
	- **Adapter Ligation**: Special Illumina adapters are ligated to the ends of the processed ds-cDNA. These adapters provide a) the necessary platform-specific sequences for the cDNA to bind to the sequencer flowcell and b) the primer sequence to initiate the sequencing reactions.
		- The adapters may also contain unique index sequences (barcodes) that allow for the identification of sequences from different samples when they are pooled together and sequenced in the same run (multiplexing, to increase throughput and reduce costs).
	- **Size Selection**: This step selects fragments of the desired size for uniform sequencing.
		- Important for optimizing NGS efficiency and data quality, as large variations in insert sizes can lead to uneven PCR amplifications and bias.
		- It also removes adapter dimers (ligated adapter pairs without cDNA insert).
		- Can be done using gel-based methods or column-based clean-up.
	- **PCR Amplification**: The size-selected cDNA fragments, now with adapters on both ends, are amplified by PCR to create a sufficient amount of material for sequencing.
	- **Final Library Preparation**: The PCR-amplified library is now ready for sequencing.
	- Each asterisk-marked step in the slide's graphic panel emphasizes critical stages in library preparation where quality control is essential. For instance, checking the size distribution after fragmentation and size selection, and verifying the successful ligation of adapters before PCR amplification. The graphic also indicates "quick ligation" suggesting that modern techniques have made this step faster, and it includes "Y-shape" adapters which are specific to Illumina's sequencing technology, indicating that the adapters have overhangs designed to interact efficiently with the sequencing platform.
	-
	- Adapter Ligation and Amplification:
		- DNA adapters are ligated to the ends of the cDNA fragments to make a cDNA-sequencing library.
		- PCR amplification (8-12 cycles) increases the quantity of cDNA fragments to be sequenced
	- Sequencing:
		- Can be single-end (SET) or paired-end (PET) sequencing
	- ### NGS sequencing of RNA-seq library
		- S26: Paired-end (PE) vs. single-end (SE) sequencing
		- S27: Slido question
		- S28: Strand-specific RNA-seq
-